posthog/api/organization.py

```diff
@@ -143,9 +146,21 @@ def create(self, validated_data, **kwargs):
 
         return user
 
-    def to_representation(self, instance):
-        serializer = UserSerializer(instance=instance)
-        return serializer.data
+    def create_team(self, organization: Organization, user: User) -> Team:
+        if self.enable_new_onboarding(user):
+            return create_demo_team(user=user, organization=organization, request=self.context["request"])
+        else:
+            return Team.objects.create_with_data(user=user, organization=organization)
+
+    def to_representation(self, instance) -> Dict:
+        data = UserSerializer(instance=instance).data
+        data["redirect_url"] = "/personalization" if self.enable_new_onboarding() else "/ingestion"
```

Comment:
Maybe reflect this new attribute in the relevant test to make sure it's behaving as expected?